pom.xml

```diff
@@ -48,13 +48,12 @@
 
     <properties>
         <jenkins.version>1.625.3</jenkins.version>
-        <jenkins-test-harness.version>1.625.3</jenkins-test-harness.version>
```

Comment:
what minimal version will be now?

Reply:
Does it matter? it is Jenkins Test Harness (test tool), not Jenkins Core

Reply:
It brings some dependencies too

Reply:
only on the test scope. The version would become 2.31. For testing of the plugin with new Jenkins core in Plugin Compat Tester it is critical to have 2.23 at least so that it supports cores without war-for-test